src/components/Admin/index.jsx

```diff
@@ -51,14 +42,31 @@ class Admin extends React.Component {
   }
 
   renderErrorMessage() {
-    const message = `There was a problem fetching dashboard data: ${this.props.error.message}`;
-    return <StatusMessage alertType="danger" message={message} />;
+    return (
+      <StatusAlert
+        className={['mt-3']}
+        alertType="danger"
+        dismissible={false}
+        dialog={
+          <div className="d-flex">
+            <div className="icon mr-2">
+              <Icon className={['fa', 'fa-times-circle']} />
+            </div>
+            <div className="message">
+              <span className="title">Unable to load overview</span>
+              <span>Try refreshing your screen ({this.props.error.message})</span>
+            </div>
+          </div>
+        }
+        open
+      />
+    );
   }
 
   renderLoadingMessage() {
     return (
-      <div className="loading">
-        <p>Loading...</p>
+      <div className="overview loading mt-3 d-flex align-items-center justify-content-center">
```

Comment:
May should wrap this as a LoadingMessage component as well. I assume we'll have loading messages on other pages so having a single place would be good - then if we ever change how we're displaying loading messages its in a single place.

If you want to leave this for now and refactor once we create the next LoadingMessage